eth-connector/src/lib.rs

```diff
@@ -213,6 +214,188 @@ impl FungibleTokenCore for EthConnectorContract {
     }
 }
 
+/// Fungible Token Trait implementation for compatibility with Engine NEP-141 methods.
+/// It's because we should ve known correct `sender_id`. In reference
```

Comment:
```suggestion
/// It's because we should have known the correctly `sender_id`. In reference
```

Reply:
* We should have a known correct `sender_id`